examples/orderbook/fulfill-collection-bid-with-nextjs/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "fulfill-collection-bid-with-nextjs",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint"
+  },
+  "dependencies": {
+    "@biom3/react": "^0.27.10",
+    "@imtbl/sdk": "latest",
+    "ethers": "^5.7.2",
+    "next": "14.2.10",
+    "react": "^18",
+    "react-dom": "^18"
+  },
+  "devDependencies": {
+    "@types/node": "^20",
+    "@types/react": "^18",
+    "@types/react-dom": "^18",
+    "eslint": "^8",
+    "eslint-config-next": "14.2.14",
+    "postcss": "^8",
+    "tailwindcss": "^3.4.1",
```

Comment:
Tailwind doesn't seem to be used at all in this example. Is it safe to be removed, along with the tailwind config file? Is postcss still needed for biom3/react or any other purpose if tailwind isn't present? If not, that could be cleaned up too, along with its config file